imgix/validators.py

```diff
@@ -0,0 +1,100 @@
+from .constants import IMAGE_MAX_WIDTH as MAX_WIDTH
+from .constants import IMAGE_ZERO_WIDTH as ZERO_WIDTH
+
+
+def validate_min_width(value):
```

Comment:
```suggestion
def validate_min_width(min_width):
```

Reply:
While I do like how explicit `min_width` is, in this particular case I feel that it's redundant (i.e. not DRY). We shouldn't repeat ourselves if we don't have to. Moreover, using this pattern could help in the future by not having multiple similarly named values within function bodies.

Reply:
It was more so a suggestion to stay consistent with `def validate_range(min_width, max_width)`. Alternative, we could also name it `width`.